mission.sqm

```diff
@@ -2740,6 +2799,942 @@ class Mission
 				};
 			};
 		};
+		class Item36
+		{
+			side="LOGIC";
+			class Vehicles
+			{
+				items=1;
+				class Item0
+				{
+					position[]={8137.624,357.56976,6775.8096};
+					id=140;
+					side="LOGIC";
+					vehicle="ACE_Vehicledamage_Enable_Cookoff";
+					leader=1;
+					lock="UNLOCKED";
+					skill=0.60000002;
+				};
+			};
+		};
+		class Item37
+		{
+			side="WEST";
+			class Vehicles
+			{
+				items=2;
+				class Item0
+				{
+					position[]={8231.8994,353.05374,7077.0728};
+					id=208;
+					side="WEST";
+					vehicle="US_Delta_Force_TL_EP1";
+					player="PLAY CDG";
+					leader=1;
+					rank="SERGEANT";
+					skill=0.60000002;
+					text="sf1";
+					init="[this, ""ASL"", ""MAIN"", ""sf_sl"", ""hq"", ""ColorRed""] call fnc_initializeUnit;";
+					description="Alpha Squad Leader";
+				};
+				class Item1
+				{
+					position[]={8235.6318,353.06238,7074.6265};
+					id=209;
+					side="WEST";
+					vehicle="US_Delta_Force_Medic_EP1";
+					player="PLAY CDG";
+					rank="CORPORAL";
+					skill=0.60000002;
+					text="sf2";
+					init="[this, ""AM"", ""RED"", ""sf_medic"", ""medic"", ""ColorRed"", 0.75, true] call fnc_initializeUnit;";
+					description="Alpha Corpsman";
+				};
+			};
+		};
+		class Item38
+		{
+			side="WEST";
+			class Vehicles
+			{
+				items=4;
+				class Item0
+				{
+					position[]={8230.207,353.0253,7068.1196};
+					id=210;
+					side="WEST";
+					vehicle="US_Delta_Force_TL_EP1";

```

Comment:
Use vanilla units for the base, and keep them all (except medic) one class

Reply:
What do you mean by vanilla units for the base?

Reply:
Actually I don't think it matters. The initial idea I was thinking of was that you just use the base 3 factions present in the framework and change them out as needed, but if you are adding new factions with their own uniforms it will be mostly the same.

If it becomes an issue with too many groups in the future this will be amended but for now dont worry